<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="INSML-H55MD6M" failures="1" tests="10" name="Default test" time="770.069" errors="0" timestamp="2022-12-17T02:20:18 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="63.477"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="124.924">
    <failure type="java.lang.AssertionError" message="Amount didn&amp;apos;t get credited to Riya">
      <![CDATA[java.lang.AssertionError: Amount didn't get credited to Riya
at org.testng.Assert.fail(Assert.java:110)
at com.paraBanking.stepDefinitions.ParaBankStepDefinitions.verifyBalanceAndTransactionTable(ParaBankStepDefinitions.java:303)
at âœ½.Verify that balance and transaction table details of Checking account is correct after receiving amount of 50 <from-to> Riya(file:///Users/snimma868/Demo/paraBanking/src/test/resources/Login.feature:27)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="72.336"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="73.416"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="72.083"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="75.223"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="71.264"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="72.359"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="72.965"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="71.819"/>
</testsuite> <!-- Default test -->
